pygeode/stats.py

```diff
@@ -125,11 +127,13 @@ def correlate(X, Y, axes=None, output = 'r2,p', pbar=None):
   # Compute correlation coefficient, t-statistic, p-value
   den = np.zeros(oview.shape, 'd')
   rho = np.zeros(oview.shape, 'd')
+  rho2 = np.zeros(oview.shape, 'd')
```

Comment:
rho2 is just rho squared and can be computed in one line below; no need for a new output array here.

Reply:
```suggestion
```